packages/wdio-utils/src/test-framework/testFnWrapper.ts

```diff
@@ -34,7 +35,8 @@ export const testFnWrapper = function (
         BeforeHookParam<unknown>,
         AfterHookParam<unknown>,
         string,
-        number
+        number,
+        string?
```

Comment:
From what I can tell, this isn't optional and is always passed

Reply:
i'm leaving this optional, since `runSpec` in `testInterfaceWrapper` calls it as well, and then we'll add an unused argument (`origFn.name` is `it`)